spec/reek/cli/command/todo_list_command_spec.rb

```diff
@@ -37,6 +37,40 @@
         result = command.execute
         expect(result).to eq(Reek::CLI::Options::DEFAULT_SUCCESS_EXIT_CODE)
       end
+
+      it 'writes a todo file' do
+        command.execute
+        expected_yaml = { 'FeatureEnvy' => { 'exclude' => ['self'] } }.to_yaml
```

Comment:
Yes, I think I do see what you mean. I've solved the problem of the obscure 'self' by providing the context value in the spec itself (hence, we're commenting on an outdated diff here).

My point o.t.o.h. was that there is no `:foo` for which `attributes_for(:foo)` gives me the expected hash. Hence, the only way I can use `attributes_for` would be something like:

```
context = attributes_for(:smell_warning)['context']
expected_yaml = { 'FeatureEnvy' => { 'exclude' => [context] } }.to_yaml
```

But then I can just do:
```
expected_yaml = { 'FeatureEnvy' => { 'exclude' => [smell.context] } }.to_yaml
```

So `attributes_for` isn't really helpful in this particular case.